doc/modules/ensemble.rst

```diff
@@ -885,8 +885,7 @@ from a sample drawn with replacement (i.e., a bootstrap sample) from the
 training set.
 
 Furthermore, when splitting each node during the construction of a tree, the
-best split is found either from all input features or a random subset of size
-``max_features``. (See the :ref:`parameter tuning guidelines
+best split is found through an exhaustive search of the features values of either all input features or a random subset of size ``max_features``. (See the :ref:`parameter tuning guidelines
 <random_forest_parameters>` for more details).
 
```

Comment:
It is better to rewrap to have less than 88 characters on a line even for text.

```suggestion
best split is found through an exhaustive search of the features values of
either all input features or a random subset of size ``max_features``.
See the :ref:`parameter tuning guidelines <random_forest_parameters>` for
more details.
```

Reply:
Ok. I kept the parentheses around the "See the..." sentence because in the document these types of sentences have parentheses around them.

Reply:
This make sense. Thanks